docs/Using-Fleet/REST-API.md

```diff
@@ -6128,6 +6132,7 @@ The returned lists are filtered based on the hosts the requesting user has acces
         "id": 3,
         "detail_updated_at": "2021-02-03T21:58:10Z",
         "label_updated_at": "2021-02-03T21:58:10Z",
+	"policy_updated_at": "2023-06-26T18:33:15Z",
```

Comment:
Indentation was off on this one when I built the docs locally; went ahead and fixed it